lib/Polaris.Class.ps1

```diff
@@ -0,0 +1,321 @@
+class Polaris {
+    
+    [Int]$Port
+    [System.Collections.Generic.List[PolarisMiddleWare]]$RouteMiddleWare = [System.Collections.Generic.List[PolarisMiddleWare]]::new()
+    [System.Collections.Generic.Dictionary[string, [System.Collections.Generic.Dictionary[string, string]]]]$ScriptBlockRoutes = [System.Collections.Generic.Dictionary[string, [System.Collections.Generic.Dictionary[string, string]]]]::new()
```

Comment:
I mentioned this in another comment but now that we're in PowerShell do the Script Blocks have to be stored as a string?

Reply:
It doesn't have to be stored as strings but they eventually would need to be converted to strings in order to be passed to the AddScript method as it only accepts strings.

https://docs.microsoft.com/en-us/dotnet/api/system.management.automation.powershell.addscript?view=powershellsdk-1.1.0